ci/ubuntu.dockerfile

```diff
@@ -7,20 +7,26 @@ ARG venv=venv
 
 RUN echo "To invalidate cache"
 
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88
```

Comment:
So, Ursa is in a different sovrin repo channel. How the installation of indy-node and plenum (from deb) should look like now? Does it mean that everyone will have to add a new repo?
This looks like a breaking change affecting all installations and upgrade procedures.

Reply:
@mikelodder7 I will let you answer this

Reply:
I was under the impression that its the same repo. Is that not the case here?

Reply:
It's `https://repo.sovrin.org/deb` for indy-plenum and indy-node and `https://repo.sovrin.org/ursa/deb` (see the next line) for Ursa.